lib/components/gene/__tests__/about-test.js

```diff
@@ -0,0 +1,37 @@
+import 'react-native'
+import React from 'react'
+import About from '../about'
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer'
+
+jest.mock('../../opaque_image_view.js', () => 'ImageView')
```

Comment:
yeah pretty much! 

Only difference is that we're looking at the DOM tree, rather than it applied to a screen - so there's no layout information outside of what CSS attributes we've declared